### DISCUSSION TITLE
How to change the logo shown in Coinbase Wallet?

### DISCUSSION THREAD

**Discussion by JorensM** [rainbow-me/rainbowkit#1514] [Q&A] [answered]
Hello, I would like to know if/how it's possible to change the logo that is shown when you connect/sign with your Coinbase Wallet.

<img width="211" alt="Screenshot 2023-08-14 at 15 51 04 (1)" src="https://github.com/rainbow-me/rainbowkit/assets/17122123/a52dbbb8-3766-460e-952a-da5347f69666">

As you can see it the image above, the logo is a bit too large and is cut off. Is it possible to change the logo?

**Comment by 0xmkh** [comment, marked as answer]
@JorensM This will be introduced in RainbowKit v2

**Reply by JorensM** [reply]
Ok, thanks
